webpack.config.js

```diff
@@ -0,0 +1,30 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: 'development',
+  entry: {
+    index: path.resolve(__dirname, './src/index.js'),
+  },
+  output: {
+    path: path.resolve(__dirname, './dist'),
+    filename: 'Running.js',
+    // clean: true,
```

Comment:
- [ ] Kindly consider removing this commented line as having commented code in our repo makes it unprofessional.